internal/server/iigointernal/monitoring.go

```diff
@@ -59,8 +59,6 @@ func (m *monitor) monitorRole(roleAccountable baseclient.Client) shared.MonitorR
 
 			message := generateMonitoringMessage(roleName, evaluationResult)
 			broadcastToAllIslands(m.iigoClients, roleAccountable.GetID(), message, *m.gameState)
-
-			m.gameState.IIGOTurnsInPower[roleName]++
```

Comment:
m.gameState.IIGOTurnsInPower[roleName] = config.IIGOTermlengths[roleName]

Reply:
or something along those lines. At some point somebody had misinterpreted and changed this to ++

Reply:
@Neelesh99

Reply:
that would be me :D why would this be in if announce {} though ?

Reply:
We removed the ++ line because it double incremented the TurnsInPower, would the suggested line do more than a double increment?

Reply:
it sets it so the responsible role would have rule obliging them to remove the monitored role from power. If the term length is 6 and we are in turn 3, this would set turns in power to 7 (and now an election is permitted and obliged to be held)

Reply:
as to why, this is what we defined the announce power to be. If there is an announcement of misconduct an election should be held

Reply:
so this should happen if the monitoringResult announced is negative right? not if announce ?

Reply:
yup